source/Google/MobileAds/ApiDefinition.cs

```diff
@@ -432,14 +471,85 @@ interface Request : INSCopying {
 		void Tag (bool forChildDirectedTreatment);
 	}
 
+	[Static]
+	interface MaxAdContentRatingConstants
+	{
+		// GAD_EXTERN GADMaxAdContentRating _Nonnull const GADMaxAdContentRatingGeneral;
+		[Field("GADMaxAdContentRatingGeneral", "__Internal")]
+		NSString General { get; }
+
+		// GAD_EXTERN GADMaxAdContentRating _Nonnull const GADMaxAdContentRatingParentalGuidance;
+		[Field("GADMaxAdContentRatingParentalGuidance", "__Internal")]
+		NSString ParentalGuidance { get; }
+
+		// GAD_EXTERN GADMaxAdContentRating _Nonnull const GADMaxAdContentRatingTeen;
+		[Field("GADMaxAdContentRatingTeen", "__Internal")]
+		NSString Teen { get; }
+
+		// GAD_EXTERN GADMaxAdContentRating _Nonnull const GADMaxAdContentRatingMatureAudience;
+		[Field("GADMaxAdContentRatingMatureAudience", "__Internal")]
+		NSString MatureAudience { get; }
+	}
+
+	// @interface GADRequestConfiguration : NSObject
+	[BaseType (typeof (NSObject), Name = "GADRequestConfiguration")]
+	interface RequestConfiguration {
+		// @property(nonatomic, strong, nullable) GADMaxAdContentRating maxAdContentRating;
+		[NullAllowed]
+		[Export ("maxAdContentRating", ArgumentSemantic.Strong)]
+		string MaxAdContentRating { get; set; }
+
+		// @property (copy, nonatomic) NSArray<NSString *> * _Nullable testDeviceIdentifiers;
+		[NullAllowed, Export ("testDeviceIdentifiers", ArgumentSemantic.Copy)]
+		string[] TestDeviceIdentifiers { get; set; }
+
+		// -(void)tagForUnderAgeOfConsent:(BOOL)underAgeOfConsent;
+		[Export ("tagForUnderAgeOfConsent:")]
+		void TagForUnderAgeOfConsent (bool underAgeOfConsent);
+
+		// -(void)tagForChildDirectedTreatment:(BOOL)childDirectedTreatment;
+		[Export ("tagForChildDirectedTreatment:")]
+		void TagForChildDirectedTreatment (bool childDirectedTreatment);
+	}
+
 	[DisableDefaultCtor]
 	[BaseType (typeof (NSError), Name = "GADRequestError")]
 	interface RequestError {
+		// extern NSString *const kGADErrorDomain __attribute__((visibility("default")));
+		[Internal]
+		[Field ("kGADErrorDomain", "__Internal")]
+		NSString _ErrorDomain { get; }
+
+		[Static]
+		[Wrap ("_ErrorDomain.ToString ()")]
+		string ErrorDomain { get; }
```

Comment:
Why not just? Error domain NSStrings do not make much sense as `string`

```
[Field ("kGADErrorDomain", "__Internal")]
NSString ErrorDomain { get; }
```

Reply:
@dalexsoto `ErrorDomain` was bound as `string`, changing this to `NSString` could cause breaking changes.